src/app/sequence-editor/sequence-editor.component.html

```diff
@@ -33,19 +33,31 @@ <h3 class="sequence-section-title">Card Preview</h3>
       <div class="flashcard-container">
         <app-flashcard [scaleFactor]="cardScaleFactor" [flashcard]="selectedFlashcard"></app-flashcard>
       </div>
-      <button mat-flat-button class="add-to-sequence-button" (click)="addToSequence($event)">Add to Sequence</button>
+      <button mat-flat-button class="add-to-sequence-button" (click)="addToSequence(selectedFlashcard)">Add to Sequence</button>
     </div>
   </div>
 
   <div class="sequence-tab-right">
-    <h3 class="sequence-section-title title-right">Sequence name 1</h3>
-    1.
-    <mat-card class="flashcard-edit">
-      <mat-card-title class="flashcard-edit-title">Term</mat-card-title>
-    </mat-card>
-    <mat-card class="flashcard-edit">
-      <mat-card-title class="flashcard-edit-title">Definition</mat-card-title>
-    </mat-card>
-    <mat-icon class="move-icon">menu</mat-icon>
+    <div class="sequence-preview" *ngIf="this.selectedSequence!=undefined; else showEmpty">
+      <mat-form-field class="sequence-name-input">
+        <input matInput type="text" [(ngModel)]="selectedSequence.name">
+      </mat-form-field>
```

Comment:
Add a mat-label with "Name of Sequence" or "Sequence Name" to this form field to make it's purpose clear.